x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/serializer.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FormInternal } from './components/phases/types';
+import { SerializedPolicy } from '../../../../common/types';
+
+export const createSerializer = (originalPolicy?: SerializedPolicy) => (
+  data: FormInternal
+): SerializedPolicy => {
+  const { _meta, ...rest } = data;
+
+  if (!rest.phases || !rest.phases.hot) {
+    rest.phases = { hot: { actions: {} } };
+  }
+
+  if (rest.phases.hot) {
+    rest.phases.hot.min_age = originalPolicy?.phases.hot?.min_age ?? '0ms';
```

Comment:
So the `rest.phases.hot.min_age` cannot be set in the form? It is simply copied over the originalPolicy or given a default value?

Reply:
Yeah, it is a bit of a weird one. This is what current serialisation behaviour gives us:
https://github.com/elastic/kibana/blob/fc5ad4d859185e344fb2e5c263f4a172d84d63ff/x-pack/plugins/index_lifecycle_management/public/application/constants/policy.ts#L63

but with a bit more indirection